setup/install/providers/appengine.md

```diff
@@ -96,7 +96,9 @@ hal config provider appengine account add my-appengine-account \
 
 You can omit the `--json-path` flag if Spinnaker does not need service account credentials.
 
-## Advanced Account Settings
+## Deploying to App Engine
+
+### Deploying from Source
 
 Spinnaker deploys to App Engine by cloning your application source code from a git repository. Unless your code
```

Comment:
I think the wording here should match that of the below section, e.g. "Spinnaker supports deploying your source code by cloning a git repo and submitting it to App Engine..."